src/test/java/org/toradocu/PrecisionRecallCommonsCollections4.java

```diff
@@ -63,21 +63,21 @@ public void testFluentIterable() throws Exception {
 
   @Test
   public void testLRUMap() throws Exception {
-    test("org.apache.commons.collections4.map.LRUMap", 0.892, 0.862, 1, 0.857, 1, 0.5);
+    test("org.apache.commons.collections4.map.LRUMap", 0.892, 0.862, 1, 1, 1, 0.5);
   }
 
   @Test
   public void testAllPredicate() throws Exception {
-    test("org.apache.commons.collections4.functors.AllPredicate", 1, 0.75, 1, 0, 1, 1);
+    test("org.apache.commons.collections4.functors.AllPredicate", 1, 0.75, 0.333, 0.333, 1, 1);
```

Comment:
In this test and in `testAnyPredicate`, the precision decreases from 1 to 0.333, meaning that Toradocu produces incorrect translations. Can you explain the problem, please?
Do you think we can address it?